pytest_selenium/driver.py

```diff
@@ -11,6 +11,9 @@
 from selenium.webdriver.firefox.firefox_binary import FirefoxBinary
 from selenium import webdriver
 
+SUPPORTED_DRIVERS = ['BrowserStack', 'Chrome', 'Firefox', 'Ie',
+                     'Opera', 'PhantomJS', 'Remote', 'SauceLabs']
```

Comment:
Personal preference to put each item on a new line. I'm also tempted to rename `Ie` as `IE` because it looks nasty.